spec/fixtures/fixtures.yml

```diff
@@ -98,7 +98,7 @@ markdown-artistic:
   hash: fb7d858f0e6b9885b5ae4a7a763888f01624ebdf
 markdown-gpl:
   key: gpl-2.0
-  matcher: exact
+  matcher: dice
```

Comment:
Curious why the matcher when from exact to dice with theoretically better normalization?

Reply:
The increased punishment for length delta exposed imperfect markdown normalization. I'm fairly certain that's what the reason is, but I'll try to confirm and fix when I have some time to dig in.

Reply:
One character fix, but it does change the hashes of many licenses.

Reply:
I benefit though is that `-diff` output will be less noisy now for some licenses.